template.yaml

```diff
@@ -7243,6 +7243,7 @@ Resources:
               - dynamodb:Query
             Resource:
               - !Sub ${AccessTokenTable.Arn}/index/*
+              - !GetAtt AccessTokenTable.Arn
```

Comment:
It's a bit nit picky, but it would be nicer to add the query action to `AllowAccessTokenTableReadAccess` rather than add the resource here, as the SID's now don't really match up with what is going on